docker-start.sh

```diff
@@ -1,15 +1,3 @@
 #!/usr/bin/env bash
 
-# shellcheck disable=SC2154,SC2086
-chia start ${service}
-
-trap "echo Shutting down ...; chia stop all -d; exit 0" SIGINT SIGTERM
-
-# shellcheck disable=SC2154
-if [[ ${log_to_file} == 'true' ]]; then
-  # Ensures the log file actually exists, so we can tail successfully
-  touch "$CHIA_ROOT/log/debug.log"
-  tail -F "$CHIA_ROOT/log/debug.log" &
-fi
-
 /prefarm-alert
```

Comment:
At this point, does this script need to exist or can this line just be the Dockerfile's CMD

Reply:
It maybe doesn't need to exist I guess.. the default docker start script will then exist in the container, but if we're overriding command, I guess it doesn't matter. I'll update